index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+
+    <header>
+        <div>
+            <nav id="head">
+                <p>Safar Ali</p>
```

Comment:
- [ ] Instead of using the` p` tag for your logo, please kindly change it with the anchor `<a>` tag.